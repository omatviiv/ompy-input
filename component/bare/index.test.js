import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Component, { thm } from 'componnt/bare';

describe('Component', () => {
  it('renders with value', () => {
    const wrapper = shallow(<Component value='test' />);
    expect(wrapper).toMatchSnapshot();
  });
  describe('theme', () => {
    it('BareInput', () => {
      const wrapper = renderer.create(<thm.BareInput />).toJSON();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
