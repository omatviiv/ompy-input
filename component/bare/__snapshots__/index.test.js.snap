// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders with value 1`] = `
<styled.input
  onChange={[Function]}
  type="text"
  value="test"
/>
`;

exports[`Component theme BareInput 1`] = `
.c0 {
  padding: 5px;
}

<input
  className="c0"
/>
`;
